src/nemos/basis.py

```diff
@@ -1075,6 +1143,11 @@ class MSplineBasis(SplineBasis):
     ----------
     [1] Ramsay, J. O. (1988). Monotone regression splines in action. Statistical science,
         3(4), 425-441.
+
+    Notes
+    -----
+    MSplines must integrate to 1 over their domain. Therefore, if the domain (x-axis) of an MSpline
+    basis is dilated by a factor of $\alpha$, the co-domain (y-axis) values will shrink by a factor of $1/\alpha$.
```

Comment:
should make it clear how this point maps to the input arguments. Something like: "For example, by default, with `bounds=(0,1)`, the maximum height is 1. However, if `bounds=(0,2)`, then the maximum height is .5". Or the like

Reply:
it's the area under the curve (like the cumsum/dx) that changes

Reply:
I'll try to explain